oct_converter/readers/e2e.py

```diff
@@ -513,3 +535,33 @@ def vol_intensity_transform(self, data: np.array) -> np.array:
         data[selection_0] = 0
         data = np.clip(data, 0, 1)
         return data
+
+    def julian_to_ymd(self, J):
```

Comment:
you asked about leaving this in - i think it is useful left in, as it will help future testing to get to the bottom of the patient birthdate mystery!